### PR TITLE
Passive scan script to alert if IBAN numbers are found in the HTTP Response

### PR DESCRIPTION
**Pull request by gmaran23**
New regular expression to identify international bank account numbers in the HTTP response body.

### CODE REVIEW

File: passive/Find IBANs.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    // first lets set up some details incase we find an IBAN, these will populate the alert later
+    var alertRisk = 1
+    var alertReliability = 3
+    var alertTitle = 'IBAN found - investigation required (script)'
+    var alertDesc = 'IBAN numbers were found'
+    var alertSolution = 'Investigate IBAN numbers found in the response, remove or mask as required'
+    var cweId = 200
+    var wascId = 0
+
+	// lets build a regular expression that can find IBAN addresses
```
Comment (thc202) [inline passive/Find IBANs.js]: Could you replace the tabs with spaces?

File: passive/Find IBANs.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	// the regex must appear within /( and )/g
+    var re = /([A-Za-z]{2}[0-9]{2}[A-Za-z]{4}[0-9]{10})/g
+
+	// we need to set the url variable to the request or we cant track the alert later
+    var url = msg.getRequestHeader().getURI().toString();
+
+	// lets check its not one of the files types that are never likely to contain stuff, like pngs and jpegs
+    var contenttype = msg.getResponseHeader().getHeader("Content-Type")
```
Comment (thc202) [inline passive/Find IBANs.js]: These variables could be camel cased.

File: passive/Find IBANs.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	var unwantedfiletypes = ['image/png', 'image/jpeg','image/gif','application/x-shockwave-flash','application/pdf']
+	
+	if (unwantedfiletypes.indexOf(""+contenttype) >= 0) {
+		// if we find one of the unwanted headers quit this scan, this saves time and reduces false positives
+    		return
+	}else{
```
Comment (thc202) [inline passive/Find IBANs.js]: The else statement could be removed.

File: passive/Find IBANs.js (hunk 1 above)
Comment (gmaran23) [reply passive/Find IBANs.js]: of course.

File: passive/Find IBANs.js (hunk 2 above)
Comment (gmaran23) [reply passive/Find IBANs.js]: camelCasing done for the variables contentType and unwantedFileTypes as well. Thanks.

File: passive/Find IBANs.js (hunk 3 above)
Comment (gmaran23) [reply passive/Find IBANs.js]: done.

**Comment by thc202** [comment]
Thank you!

**Pull request merged**